### PR TITLE
Don't panic when altering retention policy

### PR DESCRIPTION
**Pull request by otoolep**
Fixes issue #2846

### CODE REVIEW

**Comment by otoolep** [comment]
Apparent copy 'n' paste issue.

**Comment by otoolep** [comment]
@toddboom

File: meta/store.go [hunk 1]
```diff
@@ -719,7 +719,7 @@ func (s *Store) UpdateRetentionPolicy(database, name string, rpu *RetentionPolic
 	}
 
 	var replicaN *uint32
-	if rpu.Duration != nil {
+	if rpu.ReplicaN != nil {
```
Comment (otoolep) [inline meta/store.go]: This is the actual fix.

**Comment by toddboom** [comment]
+1

**Pull request merged**